bolt-sidecar/src/driver.rs

```diff
@@ -161,6 +161,16 @@ impl<C: StateFetcher, ECDSA: SignerECDSA> SidecarDriver<C, ECDSA> {
         commitment_signer: ECDSA,
         fetcher: C,
     ) -> eyre::Result<Self> {
+        // Verify the operator and validator keys with the bolt manager
+        if let Some(bolt_manager) =
+            BoltManager::from_chain(opts.execution_api_url.clone(), opts.chain.chain)
+        {
+            bolt_manager.verify_operator(commitment_signer.public_key()).await?;
+            bolt_manager
+                .verify_validator_pubkeys(&Vec::from_iter(constraint_signer.available_pubkeys()))
+                .await?;
```

Comment:
I think here we should also verify that the link between the validators and the operator is also correct

Reply:
Right, great catch thank you